packages/table/src/locator.ts

```diff
@@ -17,6 +17,8 @@ export interface ILocator {
      */
     getWidestVisibleCellInColumn: (columnIndex: number) => number;
 
+    getTallestVisibleCellInColumn: (columnIndex: number) => number;
```

Comment:
The goal of this function is rather self-evident, but let's add a description anyway since all the other functions have one.